modules/nlu/src/backend/pipelines/intents/tokens_utils.ts

```diff
@@ -0,0 +1,31 @@
+import * as sdk from 'botpress/sdk'
```

Comment:
I also created a file like this in another PR (https://github.com/botpress/botpress/pull/2142/files)

> modules/nlu/src/backend/tools/token-utils.ts

It contains, among other things, the function makeTokens.

Should we merge both files and if so, where do you think this file belongs?

Reply:
Yes we probably should, somewhere in the pipelines? ;) Probably in the `intents` folder. But we can probably put it in the pipeline root also.

Reply:
Ok, lets keep this location.

> modules/nlu/src/backend/pipelines/intents/tokens_utils.ts